speech/google/cloud/speech/client.py

```diff
@@ -199,6 +223,98 @@ def sync_recognize(self, sample, language_code=None,
         else:
             raise ValueError('result in api should have length 1')
 
+    def stream_recognize(self, sample, language_code=None,
+                         max_alternatives=None, profanity_filter=None,
+                         speech_context=None, single_utterance=False,
+                         interim_results=False):
+        """Streaming speech recognition.
+
```

Comment:
Since Sphinx doesn't have a `yields` tag, can you put this in the docstring:

```python
        """Streaming speech recognition.

        Yields :class:`WhateverTypeItYields` some prose here to describe what happens...

        ...
        """